config/routes.rb

```diff
@@ -2,7 +2,7 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   get 'application/home'
 
-  resources :images
+  resources :images, only: %i[index new create show]
 
-  root 'application#home'
```

Comment:
Is the `application#home` action used anymore? Could it be deleted?

Reply:
No, and yes.  Will update.